.github/workflows/ci.yml

```diff
@@ -74,11 +74,16 @@ jobs:
         with:
           ruby-version: '3.0'
           bundler-cache: true
+      - name: Start squid
```

Comment:
These changes to `.github/workflows/ci.yml` will be overwritten by modulesync though, right?

Reply:
It was manually added in the example I was sent and has not since been overwritten by modulesync: 
https://github.com/voxpupuli/puppet-gitlab_ci_runner/commit/674b47e9e724d58a71c031024197b496e0b52fb3

Reply:
My bad.  Looks like it should have been, but @bastelfreak amended the first commit https://github.com/voxpupuli/puppet-gitlab_ci_runner/commit/5aab2653f5289446df1e72f24fb97a388fad7f7d into https://github.com/voxpupuli/puppet-gitlab_ci_runner/commit/048618a3e98ac9cbbef44faa7f8cd41600af6649

It's all change again since https://github.com/voxpupuli/modulesync_config/commit/4dc9a816f2049b257915e633639da949260f56e7

@ekohl Is there a better way to handle this other than setting the file as `unmanaged` in `.sync.yaml`?

Reply:
Done!